addons/test_mail/tests/test_mail_multicompany.py

```diff
@@ -388,6 +389,43 @@ def _mock_filter_access_rules(*args, **kwargs):
                     other_activity_group = next(ag for ag in activity_groups if ag['model'] == 'mail.activity')
                     self.assertEqual(other_activity_group["total_count"], 5)
 
+    def test_cross_company_partner_mention(self):
```

Comment:
Don't forget alphabetical ordering of tests inside a class, as it is the execution order (that way execution order = definition order and it eases finding tests when having several of them to update)